layouts/black-lives-matter.hbs

```diff
@@ -0,0 +1,230 @@
+      <li>Sean Reed</li>
+      <li>Stephon Clark</li>
+      <li>Steven Demarco Taylor</li>
+      <li>Tamir Rice</li>
+      <li>Terence Crutcher</li>
+      <li>Terrance Franklin</li>
+      <li>Terrence Sterling</li>
+      <li>Tony McDade</li>
+      <li>Tony Robinson</li>
+      <li>Trayvon Martin</li>
+      <li>Victor White III</li>
+      <li>Walter Scott</li>
+      <li>Wendell Allen</li>
+      <li>William Green</li>
+      <li>Willie McCoy</li>
+      <li>Yassin Mohamed</li>
+      <li>... <a href="https://mappingpoliceviolence.org/unarmed">and</a> <a href="https://mappingpoliceviolence.org/unarmed2014">countless</a> <a href="https://apps.npr.org/documents/document.html?id=6933593-NPR-CodeSwitch-Saytheirnameslistv3">more</a> <a href="https://ebwiki.org/">names</a> not listed here whose lives were taken by violence and brutality.</li>
+    </ul>
+    <p>We've listed the names of the Black folks who have been murdered by police brutality in the U.S. dating back to 2009, both because the U.S. is the origin of the Black Lives Matter movement and currently the heart of the resistance.</p>
+
+    <p>With that said, we fully recognize that white supremacy and police brutality is a global problem. While we have not listed all of Black people who have lost their lives to police murder globally here, their lives mattered.</p>
+  </section>
+
+  <section>
+    <p>Where you can donate to with your wallet:</p>
+    <ul>
+      <li>Donate to the <a href="https://www.gofundme.com/f/georgefloyd">Official George Floyd Memorial Fund</a>.</li>
+      <li>Donate to the <a href="https://www.gofundme.com/f/i-run-with-maud">I Run With Maud</a> fundraiser for Ahmaud Arbery.</li>
+      <li>Debate to the <a href="https://www.gofundme.com/f/justice-for-mike-ramos">Justice for Mike Ramos</a> fundraiser.</li>
+      <li>Find and donate to bail fund via the <a href="https://www.communityjusticeexchange.org/nbfn-directory">National Bail Fund Network</a>.</li>
+      <li><a href="https://www.facebook.com/reclaimtheblock/">Reclaim the Block</a> has published a <a href="https://docs.google.com/document/d/1yLWGTQIe3967hdc9RSxBq5s6KKZHe-3_mWp5oemd7OA/preview?pru=AAABcpUiX3k*Y6Q4I6UBtkH3lLz9GVLg0A">Google Doc of organizations to donate to.</a></li>
+    </ul>
+    <p>We will be updating this list periodically.</p>
+  </section>
+
+  <section id="our-support">
+
+    <p>It would be wrong for us to ignore the unearned privilege that exists in Node.js: much of the project leadership is white, and a majority are men. We are and have been systematically complicit in perpetuating the issues that led to where we are now. We can't change that, but we can change how we proceed moving forward. We are dedicated to being a part of the solution by actively centering the Black community with our collective platforms to uplift Black voices, share projects founded and built by Black community members, and by financially supporting work from Black creators.</p>
```

Comment:
```suggestion
    <p>It would be wrong for us to ignore the unearned privilege that exists in Node.js: much of the project leadership is white, and a majority are men. We are and have been systematically complicit in perpetuating the issues that led to where we are now. We can't change that, but we can change how we proceed moving forward. We are dedicated to being a part of the solution by actively centering the Black community with our collective platforms to uplift Black voices, share projects founded and built by Black community members, and actively evaluating our progress and making necessary changes as needed.</p>
```

This was originally drafted as commitments from individuals but it reads to me as a commitment from our organization. I don't currently see a way that we can clearly follow through on the commitment to financially support work by black creators. While I 100% support this action, I do not want us committing to action that we do not have a clear path to following through on.

What I had not seen in this document yet was a clear commitment to continuing to evaluating our effort, which seems like a solid replacement.

To be clear, we should figure out how to financially support folks, and once we have a clear action plan add this commitment back.

Reply:
This is a great point. I also think both shouldering the responsibility overtly while providing room for feedback. Probably needs a less foggy brain to look at, but general review.

```suggestion
    <p>It would be wrong for us to ignore the unearned privilege that exists in Node.js: much of the project leadership is white, and a majority are men. We are and have been systematically complicit in perpetuating the issues that led to where we are now. We can't change that, but we can change how we proceed moving forward. We are dedicated to being a part of the solution by actively centering the Black community with our collective platforms to uplift Black voices, share projects founded and built by Black community members, and actively hold ourselves accountable our promise and make changes as needed. While it is our responsibility and it is not the work of the community to hold us a accountable, we welcome you to share ways that we can do better [tactical details].</p>
```

Reply:
Slight tweak of what @tatianamac suggested for more clarity (after further discussion with her):

```suggestion
    <p>It would be wrong for us to ignore the unearned privilege that exists in Node.js: much of the project leadership is white, and a majority are men. We are and have been systematically complicit in perpetuating the issues that led to where we are now. We can't change how we've built out the project's leadership to date, but we can focus on changing how we build it moving forward. We are dedicated to being a part of the solution by actively centering the Black community with our collective platforms to uplift Black voices, share projects founded and built by Black community members, and actively hold ourselves accountable our promise and make changes as needed. While it is our responsibility and it is not the work of the community to hold us a accountable, we welcome you to share ways that we can do better [tactical details].</p>
```

Specifically,

```
We can't change that, but we can change how we proceed moving forward.
```
becomes
```
We can't change how we've built out the project's leadership to date, but we can focus on changing how we build it moving forward.
```

Reply:
```suggestion
    <p>It would be wrong for us to ignore the unearned privilege that exists in Node.js: much of the project leadership is white, and a majority are men. We are and have been systematically complicit in perpetuating the issues that led to where we are now. We can't change how we've built out the project's leadership to date, but we can focus on changing how we build it moving forward. We are dedicated to being a part of the solution by actively centering the Black community with our collective platforms to uplift Black voices, share projects founded and built by Black community members, and actively hold ourselves accountable to our promises and make changes as needed. While it is our responsibility and it is not the work of the community to hold us a accountable, we welcome you to share ways that we can do better by emailing blacklivesmatter@nodejs.org or reaching out individually to any of our project leaders.</p>
```
> and actively hold ourselves accountable our promise and make changes as needed
to
> and actively hold ourselves accountable to our promises and make changes as needed

> [tactical details]
to
> by emailing blacklivesmatter@nodejs.org or reaching out individually to any of our project leaders

Reply:
```suggestion
    <p>It would be wrong for us to ignore the unearned privilege that exists in Node.js: much of the project leadership is white, and a majority are men. We are and have been systematically complicit in perpetuating the issues that led to where we are now. We can't change how we've built out the project's leadership to date, but we can change how we build it moving forward. We are dedicated to being a part of the solution by actively centering the Black community with our collective platforms to uplift Black voices, share projects founded and built by Black community members, and actively hold ourselves accountable to our promises and make changes as needed. While it is our responsibility and it is not the work of the community to hold us accountable, we welcome you to share ways that we can do better by emailing blacklivesmatter@nodejs.org or reaching out individually to any of our project leaders.</p>
```

Reply:
@MylesBorins my update of `[tactical details]` is slightly more complex but we can tweak that after shipping if we want. LGTM :shipit: